lib/mongoid/persistable/updatable.rb

```diff
@@ -27,11 +27,7 @@ def update_attribute(name, value)
         as_writable_attribute!(name, value) do |access|
           normalized = name.to_s
           setter = "#{normalized}="
-          if respond_to?(setter)
-            send(setter, value)
-          else
-            write_attribute(access, value)
-          end
+          process_attribute(normalized, value)
```

Comment:
@johnnyshields Yes you are right. the line ```setter = "#{normalized}="``` is no longer needed here. My bad. I have removed that line and updated PR. Please check this now.